### ISSUE TITLE
Body background

### ISSUE THREAD

**Issue by cosme12** [delight-im/OpenSoccer#271]
I think we need a body backrgound insteed of the "old" (ugly?) gray we have right now. Maybe some stipped backrgound or with drawings. Any suggestion?

**Comment by mwaclawek** [comment]
Good idea :)

We could either use some green grass background (i.e playing field).

Or we use some pattern from this page: http://thepatternlibrary.com/ Just scroll down to see the single patterns and right-click to choose "Show background graphic" to get the file. We could discuss some patterns here.

Anyway, I think the background must be some repeating pattern (as on that site) because that is better for scaling (desktop + mobile).

**Comment by cosme12** [comment]
Right, I agree we should use some little square and repeat it. But I have no idea which one could look ok.

**Comment by mwaclawek** [comment]
Personally, I like all of the following:

 * http://i.imgur.com/6F6eAzr.png
 * http://i.imgur.com/2Nrxex6.jpg
 * http://i.imgur.com/hCrSRTN.png
 * http://i.imgur.com/KCa8KHY.png
 * http://i.imgur.com/IuijDCu.png
 * http://i.imgur.com/HMsJMTw.png

... though not necessarily for this kind of game and the current design ;)

Something like this might be more appropriate:

 * http://www.brusheezy.com/textures/20185-seamless-green-grass-textures
 * https://www.flickr.com/photos/jcwighton/3004322071/
 * http://j-m-s.deviantart.com/art/Grass-is-Grass-22138469

And then there's also:
http://subtlepatterns.com/

**Comment by cosme12** [comment]
What about using the background we have here?
![ic_launcher](https://cloud.githubusercontent.com/assets/6611118/7662363/cdd8e6fa-fb31-11e4-86a4-9ffa94e35612.png)

**Comment by mwaclawek** [comment]
Well, those two colors in the foreground are already prominently shown in our header, right? Having them (one of them) in the background as well -- wouldn't that be too much? Apart from that, weren't you thinking about something more "exciting" than just "color"? :)

**Comment by cosme12** [comment]
Yes, you are right. To many options I dont know what to choose :S

**Comment by cosme12** [comment]
What about something like this?
![screenshot](https://cloud.githubusercontent.com/assets/6611118/7737096/feefc060-ff1e-11e4-9647-4e1d36e7a293.png)

**Comment by mwaclawek** [comment]
Not bad :) But I think this cannot be tiled (e.g. in 64x64 pieces) so you need a big 1920x1080 image or something like this.

The graphics from the sites that I linked to (patterns/tiles) *could* be.

**Comment by FridgeProduction** [comment]
@cosme12 From where did you get that?

**Comment by cosme12** [comment]
I think i did it myself. I dont remember. It was a long time ago :P 
Here is it. Its just a vertical strip.
![background](https://cloud.githubusercontent.com/assets/6611118/11874597/5d74ae0c-a4bf-11e5-8938-14395c134496.png)

**Comment by FridgeProduction** [comment]
Thanks <3

Where do you have to place it with what name on the server?

**Comment by cosme12** [comment]
1) Open the ```images``` folder and paste the background strip there (with the name ```background.png```)
2) Go to ```images/Refresh.php``` line 42
3) Remove that line and replace it with ```background: rgba(0, 0, 0, 0) url("/images/background.png") repeat fixed 0 0;```

I think that should be enough.

**Comment by FridgeProduction** [comment]
I fixed it in another way,
But thanks anyway.
